modules/installation-extracting-infraid.adoc

```diff
@@ -73,10 +97,10 @@ templates contain references to this infrastructure name, so you must extract
 it.
 endif::azure[]
 
-ifdef::vsphere[]
+ifdef::vsphere,vmc[]
```

Comment:
I am not seeing this paragraph for this section in this assembly, should it be there: https://deploy-preview-29632--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-network-customizations-user-infra.html#installation-extracting-infraid_installing-vmvc-network-customizations-user-infra

Per the conditionals above it seems like it should, but not sure.